### ISSUE TITLE
Intro disappears when typing in search query

### ISSUE THREAD

**Issue by ghost** [klaftertief/elm-search#4]
I don't think the intro should disappear when you're typing in search queries. For instance, I could want to type in one of the example queries, with or without slight changes, so I want to be able to see them as I type. But the main thing is that when it disappears it looks like the results are going to come up, but then they don't and I've waited for nothing.

I saw your comment about UX and decided to start here. I'm developing my own application and I totally sympathise with not wanting to make something public until the UX is right. But I do believe it's better to have users feeding back on UX, rather than trying to imagine use cases. It's hard to complain if it's functionality you can't get anywhere else. So I hope this helps!

**Comment by klaftertief** [comment]
The intro now stays, you might need to refresh your browser to get the new script.
And thanks for the feedback, I'm happy to see that people are using it, even though I would not call it finished.

**Comment by prikhi** [comment]
After my changes, you have to submit an empty search to get the intro text back. I was thinking I could add a Message triggered by clicking the logo that would clear the search & show the intro text again. @klaftertief let me know if you'd accept that or have any feedback.

> I'm happy to see that people are using it, even though I would not call it finished.

Coming from Haskell and Hoogle, it's super useful way to discover common functions with different names(like `const` == `always`) and to figure out what packages some functions are in(like `Maybe a -> Bool`).

And due to the nature of the app, all your users are Elm users that can potentially contribute back to the project.

If you have any other ideas for fleshing out the app to where you'd consider it "finished", feel free to make some issues & I'll see if I can tackle them.

**Comment by klaftertief** [comment]
Yep, I noticed the behaviour to get back the intro text. I have to think about this page (or more general some Help UI) as there searching by name or type are not the only available searches. For now it's ok.

I'll be talking about elm-search at the next [Elm Remote Meetup](https://www.bigmarker.com/remote-meetup/Elm-Remote-Meetup-4) on September 21 and I'll set up a proper readme and issues until then. This is the other main thing to do for a proper announcement.

I'm sick a the moment, so you sadly have to wait a little for a more detailed feedback.

**Comment by prikhi** [comment]
All good, please take your time.
